src/bridge/bridge.c

```diff
@@ -682,12 +685,33 @@ main (int argc,
   signal (SIGABRT, cockpit_test_signal_backtrace);
   signal (SIGSEGV, cockpit_test_signal_backtrace);
 
-  /* In case we are run via sshd and we have journald, make sure all
-   * logging output ends up in the journal on *this* machine, not sent
-   * back to the client.
-   */
-  if (g_getenv ("SSH_CONNECTION") && !g_log_writer_is_journald (2) && ! isatty(2))
+  if (g_strv_contains ((const gchar **) argv, "--privileged"))
```

Comment:
Thinking out loud: why would we do this only for privileged peers? Why not do it always? One answer: because not all things that start cockpit-bridge understand the send-stderr command.

Reply:
So I think about the potential implications of a weird remote host telling us to send it stderr over ssh, for example.  It just doesn't make sense, so I fence it in for the time being to what we actually need it for.

If we want to expand it later, that will be easy.

Reply:
Makes sense.